lib/constants.go

```diff
@@ -1103,9 +1118,18 @@ var DeSoMainnetParams = DeSoParams{
 	// The number of epochs before an inactive validator is jailed
 	DefaultJailInactiveValidatorGracePeriodEpochs: uint64(48),
 
+	// The rate of growth of the fee bucket ranges.
+	DefaultFeeBucketRateMultiplierBasisPoints: uint64(1000),
+
+	// The rate of the failing transaction's fee used in BMF calculations.
+	DefaultFailingTransactionBMFRateBasisPoints: uint64(2500),
+
 	ForkHeights:                 MainnetForkHeights,
 	EncoderMigrationHeights:     GetEncoderMigrationHeights(&MainnetForkHeights),
 	EncoderMigrationHeightsList: GetEncoderMigrationHeightsList(&MainnetForkHeights),
+
+	MaxMempoolPosSizeBytes:        3 << 30, // 3Gb
```

Comment:
Let's make these flags. See previous comment.

Reply:
Yup, it's a flag now. It's still in DeSoParams tho, as opposed to being passed as part of NewMempool() signature. It seems cleaner this way